src/Sulu/Bundle/CategoryBundle/Domain/Event/CategoryCreatedEvent.php

```diff
@@ -72,18 +72,23 @@ public function getResourceId(): string
         return (string) $this->category->getId();
     }
 
-    public function getResourceLocale(): ?string
+    public function getResourceLocale(): string
     {
         return $this->locale;
     }
 
     public function getResourceTitle(): ?string
     {
-        $translation = $this->category->findTranslationByLocale($this->locale);
+        $translation = $this->category->findTranslationByLocale($this->getResourceTitleLocale());
```

Comment:
not sure if this makes it harder to understand what is happening, but dont have a too strong opinion 🙂

Reply:
I just want to make sure, that the locale of the `resourceTitle` is identical to the result of `DomainEvent::getResourceTitleLocale()`